gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
```

Comment:
Like most Python project we follow the PEP8 coding style.
The `pep8` tool tells you what you need to fix, on this line you should write `ax=None` instead of `ax = None`, i.e. remove the spaces around the equals sign:
```
$ pep8 gammapy/time/plot.py 
gammapy/time/plot.py:9:1: E302 expected 2 blank lines, found 1
gammapy/time/plot.py:32:1: E302 expected 2 blank lines, found 1
gammapy/time/plot.py:32:57: E251 unexpected spaces around keyword / parameter equals
gammapy/time/plot.py:32:59: E251 unexpected spaces around keyword / parameter equals
gammapy/time/plot.py:63:80: E501 line too long (91 > 79 characters)
gammapy/time/plot.py:81:80: E501 line too long (92 > 79 characters)
gammapy/time/plot.py:84:80: E501 line too long (81 > 79 characters)
gammapy/time/plot.py:86:80: E501 line too long (89 > 79 characters)
gammapy/time/plot.py:89:80: E501 line too long (90 > 79 characters)
gammapy/time/plot.py:91:80: E501 line too long (82 > 79 characters)
gammapy/time/plot.py:102:80: E501 line too long (101 > 79 characters)
gammapy/time/plot.py:104:80: E501 line too long (120 > 79 characters)
gammapy/time/plot.py:105:80: E501 line too long (120 > 79 characters)
gammapy/time/plot.py:146:80: E501 line too long (90 > 79 characters)
gammapy/time/plot.py:147:80: E501 line too long (98 > 79 characters)
gammapy/time/plot.py:150:80: E501 line too long (99 > 79 characters)
gammapy/time/plot.py:151:18: E265 block comment should start with '# '
gammapy/time/plot.py:153:80: E501 line too long (100 > 79 characters)
gammapy/time/plot.py:154:80: E501 line too long (117 > 79 characters)
gammapy/time/plot.py:155:80: E501 line too long (83 > 79 characters)
gammapy/time/plot.py:156:80: E501 line too long (102 > 79 characters)
gammapy/time/plot.py:157:80: E501 line too long (119 > 79 characters)
gammapy/time/plot.py:158:80: E501 line too long (84 > 79 characters)
gammapy/time/plot.py:165:5: E265 block comment should start with '# '
gammapy/time/plot.py:167:14: W292 no newline at end of file
```
You can install and run the `autopep8` tool to automatically fix those.
One of the few exceptions is that sometimes it makes sense to have lines > 80 chars, so my suggestion would be to always run `git diff` before accepting all `autopep8` suggestions.

You could also give this Python IDE a try: https://www.jetbrains.com/pycharm/
Open-source projects like Gammapy get free licenses for team members, so if you want one, please email me.